CHANGELOG.md

```diff
@@ -6,15 +6,16 @@
 
 ### Breaking Changes
 
-_None_
+* Clients adopting the new version.properties will need to implement a gradle task to update the version.properties file named `updateVersionProperties`
```

Comment:
I'm unsure if that's a breaking change, given that you made all the efforts to make this release backwards compatible with projects not updating this new feature… So I think this not can just go in the New Features section as part of the bullet point about new `version.properties` support.

Otherwise if you put this in Breaking Changes, this will mean a major version bump to `2.0.0`, and thus client apps not auto-updating to the release toolkit automatically because of the major version bump won't be semantically compatible with `~> 1.0`… even if in practice nothing will break if they update because the change is backwards compatible…

Reply:
ok sounds good!

Reply:
I had somewhat considered it a breaking change because it's somewhat a requirement for completely adopting the approach but I think your reasoning here is better